README.md

```diff
@@ -4,20 +4,28 @@
 
 This repository is used with `ddev get platformsh/ddev-platformsh` to get a rich integration between your checked-out Platform.sh project and [DDEV](https://github.com/drud/ddev).
 
-## Installing
+## Using with a Platform.sh project
+1. Make sure you have [DDEV v1.21.1+ installed](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/)
+2. Clone your project repository (e.g. `platform get <projectid>`)
+3. `cd` into your project directory
+4. Run `ddev config` and answer the questions as appropriate
+5. Run `ddev get platformsh/ddev-platformsh` and answer the questions as appropriate
+6. Run `ddev start`
+7. Run `ddev pull platform` to retrieve a copy of the database and contents from the project's file mounts from the environment you entered in step #5
+   1. If you only want to retrieve the database (skipping the file mounts), add the `--skip-files` flag to the `ddev pull platform` command
+
+## Notes
 
 * If your local project has a different database type than the upstream (Platform.sh) database, it will conflict, so please back up your database with `ddev export-db` and `ddev delete -y` before starting the project with new config based on upstream.
-* This requires DDEV v1.21.1+.
-* `ddev get platformsh/ddev-platformsh` and `ddev restart`
 * Your experience is super-important: Please let us know about how it went for you in any of the [DDEV support venues](https://ddev.readthedocs.io/en/latest/#support-and-user-contributed-documentation)
 
 ## What does it do right now?
 
 * Takes your checked-out Platform.sh project and configures DDEV based on that information.
-  * PHP and Database version
-  * hooks are converted to DDEV post-start hooks
-  * A working `ddev pull platform` integration with all mounts is created.
-  
+    * PHP and Database version
+    * hooks are converted to DDEV post-start hooks
+    * A working `ddev pull platform` integration with all mounts is created.
```

Comment:
We should mention that we support configured services, and which ones are supported.